### ISSUE TITLE
Zombies

### ISSUE THREAD

**Issue by hozdenoren** [JuliaDynamics/Agents.jl#539]
Hi, I am playing around with the Zombies example. However, I get the following error:

```
ERROR: LoadError: MethodError: no method matching generate_point_in_bounds(::Type{Random._GLOBAL_RNG}, ::OpenStreetMapX.MapData)
Closest candidates are:
  generate_point_in_bounds(::AbstractRNG, ::OpenStreetMapX.MapData) at C:\Users\hozdenor\.julia\packages\OpenStreetMapX\iGpak\src\routing.jl:400
Stacktrace:
 [1] generate_point_in_bounds(m::OpenStreetMapX.MapData)
   @ OpenStreetMapX C:\Users\hozdenor\.julia\packages\OpenStreetMapX\iGpak\src\routing.jl:407     
 [2] random_position(model::AgentBasedModel{OpenStreetMapSpace, Zombie, typeof(fastest), Nothing})
   @ Agents C:\Users\hozdenor\.julia\packages\Agents\YADxD\src\spaces\openstreetmap.jl:354        
 [3] initialise(; map_path::String)
   @ Main c:\Users\hozdenor\.julia\packages\Agents\YADxD\examples\zombies.jl:28
 [4] initialise()
   @ Main c:\Users\hozdenor\.julia\packages\Agents\YADxD\examples\zombies.jl:25
 [5] top-level scope
   @ c:\Users\hozdenor\.julia\packages\Agents\YADxD\examples\zombies.jl:99
in expression starting at c:\Users\hozdenor\.julia\packages\Agents\YADxD\examples\zombies.jl:99
```

OS: Windows
Julia v1.6.2
Agents v4.0.1

**Comment by AayushSabharwal** [comment]
v4.0.1 is a fairly old version of the library. Does this error still occur on the latest version?

**Comment by Libbum** [comment]
If you would like to stay on 4.0.1, you can take a look at the example code from [that version](https://juliadynamics.github.io/Agents.jl/v4.0/examples/zombies/) on the documentation page.

**Comment by hozdenoren** [comment]
Thanks, updating the package to 4.5.1 and using the example on Github solved the issue. 

I actually had just clean installed the package yesterday and assumed I was on the latest version. I am guessing it was because I was using Pluto. I just checked what version Pluto installs on Binder, and it installs 4.4.1 when you run "using Agents" in a Pluto notebook. Either I misreported 4.4.1 as 4.0.1, or something else caused Pluto to revert to 4.0.1.

As a side note, previously I used ".julia\packages\Agents\YADxD\examples\zombies.jl" that was downloaded as part of the version I had.
